lib/client/lambda-runner.ts

```
@@ -78,6 +96,17 @@ export class LambdaRunner {
 
     public stop (onStop?: () => void): void {
         this.watcher.close();
+
+        if (this.debug) {
+            let request: IncomingMessage = null;
+            for (request of this.requests) {
+                try {
+                    request.socket.end();
+                } catch (e) {
+
```

Does this error need to be printed or just swallowed?